docs/observability/Observability.md

```diff
@@ -7,4 +7,25 @@ Every component has been integrated with the above library and produces traces a
 Metrics are can be scraped via the default HTTP component at the `/metrics` route for Prometheus.  
 Traces will be sent to a Jaeger agent, which can be setup through environment variables mentioned in the config section.    
 Sane defaults are applied for making the use easy.  
-The `component` and `client` packages implement capturing and propagating of metrics and traces.
\ No newline at end of file
+The `component` and `client` packages implement capturing and propagating of metrics and traces.
+
+## Prometheus Exemplars
+
+After a release of [Grafana Tempo](https://grafana.com/oss/tempo/) and
+[Prometheus Exemplars](https://grafana.com/docs/grafana/latest/basics/exemplars/)
+it became possible to move seamlessly between metrics, logs, and traces.
```

Comment:
I think this can be rephrased as any OpenTracing compatible tracing system (Jaeger and Tempo so far only AFAIK) can work with Prometheus Exemplars, not just Tempo.
The second part regarding `seamless integration between metrics, logs and traces` is a Grafana specific feature.

Reply:
rephrased according to the comment, removed the second part.